src/qt/qt_main.cpp

```diff
@@ -17,6 +17,10 @@
  *          Copyright 2021-2022 Cacodemon345
  *          Copyright 2021-2022 Teemu Korhonen
  */
+
+#ifdef _WIN32
+#define UNICODE 1
+#endif
```

Comment:
No need, just don't use the tchar macros and call the function with the appropriate suffix explicitly if you're hardcoding narrow/wide strings